### ISSUE TITLE
Custom Fields

### ISSUE THREAD

**Issue by slightlychilled** [croogo/croogo#961]
It would be a good option to pre-define a set up custom fields based on node-type. So, when a node is created based on type those custom fields are automatically displayed, ready for input, and preferably under the main body and excerpt fields.

**Comment by rchavik** [comment]
I agree this would be a good feature to have, but I won't have time to work on this at this time.

I'd welcome a PR.

**Comment by slightlychilled** [comment]
I'd also suggest that we move the custom fields to the middle column below excerpt. IMHO it would be more user friendly.
